utils/ntfs-3g/Makefile

```diff
@@ -215,8 +215,8 @@ endef
 
 define Package/ntfs-3g-utils/install
 	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/ntfs-3g.secaudit $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/ntfs-3g.usermap $(1)/usr/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/man/man8/ntfssecaudit.8 $(1)/usr/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/man/man8/ntfsusermap.8 $(1)/usr/bin
```

Comment:
These 2 utilities have been merged into ntfsprogs. See the 5th item in the change list above. This change is in error.